stdlib/stdlib.mli

```diff
@@ -361,7 +361,7 @@ external ( * ) : int -> int -> int = "%mulint"
 
 external ( / ) : int -> int -> int = "%divint"
 (** Integer division.
-   Raise [Division_by_zero] if the second argument is 0.
+   @raise Division_by_zero if the second argument is 0.
```

Comment:
This wants to go below the paragraph (i.e. at the end of the documentation comment)